src/site/content/en/fast/reduce-network-payloads-using-text-compression/index.md

```diff
@@ -156,3 +158,35 @@ const app = express();
 
 app.use(express.static('public'));
 </pre>
+
+## Effects on Core Web Vitals
+
+Depending the type of resource, both compression and minification can have a variety of effects on Core Web Vitals. Since both minification and compression reduce the payload size of the delivered resource, it's important to understand what Core Web Vitals are affected by these performance optimizations.
+
+### CSS
+
+When CSS resources are compressed and minified, a number of performance improvements may be observed in your Core Web Vitals metrics.
+
+- Where [First Contentful Paint (FCP)](/fcp/) is concerned, making your CSS as small as it can possibly be means that the browser will take less time to download the resource. This means that parsing of CSS can occur sooner, which should move up the rendering step.
+- Where [Largest Contentful Paint (LCP)](/lcp/) is concerned, it largely depends. If your LCP resource is specified in a CSS `background-image` property, making your CSS smaller means that the potential LCP resource can be discovered more quickly. However, LCP candidates that come from a CSS `background-image` property are not always optimal for LCP, as external CSS resources [don't benefit from the browser preload scanner](/preload-scanner/#css-background-images). If you can't move an LCP image out of CSS and into your HTML, then you should be preloading that image resource.
```

Comment:
Again "it largely depends". Does that mean you'd suggest NOT compressing or minifying for some cases? No.

I'd also merge this, concentrate less on the `background-image` example and concentrate instead on the render-blocking aspect. Something like this:

```suggestion
CSS is render-blocking so the quicker it can be downloaded and parsed, the quicker these renders can happen which can help both [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/).
```

Yeah an image may still need to be fetch for LCP but not for text ones. Plus images (without `fetchpriority="high"`) are not even fetched until the browser gets through the critical resources. So I think above shorter version if fine.